Testing/CMakeLists.txt

```diff
@@ -13,14 +13,16 @@ set( testing_HEADERS
 )
 
 set( testing_SOURCES
-    TestControlAccess.cc
+#    TestControlAccess.cc
```

Comment:
Why is Control Access test not compiled?

Reply:
I never really updated TestControlAccess to be useful tests.  It has some older code that's not applicable to the current setup.  I will do this before merging this branch.